Products/ZenModel/MaintenanceWindow.py

```diff
@@ -239,21 +240,24 @@ def makeInt(v, fieldName, minv=None, maxv=None, acceptBlanks=True):
 
         oldAuditData = self.getAuditData()
         msgs = []
-        # startHours, startMinutes come from menus.  No need to catch
-        # ValueError on the int conversion.
-        startHours = int(startHours)
-        startMinutes = int(startMinutes)
         self.enabled = bool(enabled)
-        import re
-        try:
-            month, day, year = re.split('[^ 0-9]', startDate)
-        except ValueError:
-            msgs.append("Date needs three number fields")
-        day = int(day)
-        month = int(month)
-        year = int(year)
-        if not msgs:
-            t = time.mktime((year, month, day, startHours, startMinutes,
+        if startDateTime:
+            t = startDateTime
```

Comment:
There should be code here to make sure that `t` references an integer type.  Something like:
```
    t = int(startDateTime)
```
This code will work whether `startDateTime` is an `int` or a string representing an integer.